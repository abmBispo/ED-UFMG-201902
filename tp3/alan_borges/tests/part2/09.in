32
olhe e melhor depois do cha logo nao e nada mau mamae assusta se por tudo nao e cousa de cuidado nao e molestia nao senhora e isso e volta de constipacao
c e
c nao
c de
c tudo
c volta
c assusta
c cha
c constipacao
c cousa
c cuidado
c depois
c do
c isso
c logo
c mamae
c mau
c melhor
c molestia
c nada
c olhe
c por
c se
c senhora
