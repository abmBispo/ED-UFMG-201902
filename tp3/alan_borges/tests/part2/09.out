111
001
0100
0000
0001
01010
01011
01100
01101
01110
01111
10000
10001
10010
10011
10100
10101
10110
10111
11000
11001
11010
11011
